R/preview-standalone.R

```diff
@@ -436,3 +436,108 @@ st_image_show <- function(path,
   )
   return(ans)
 }
+
+#' Render and save a table in png or pdf format
+#'
+#' @param x an stable object.
+#' @param file the final output file name; if `file` has `.tex` extension, the
+#' extension will be replaced with the value in `format`; if provided as a
+#' complete path, directory information will be used as `dir` (see below).
+#' @param stem used to form the final output file name; this argument will
+#' override what is provided in `file`.
+#' @param format the output file format.
+#' @param dir the final output directory; will be overwritten if `file` is
```

Comment:
Maybe "overridden" instead of "overwritten"? I'm worried that someone reading this quickly will interpret it as the directory itself being overwritten rather than `dir` being set to the value of `file`s directory.

Reply:
Thanks; good suggestion ... I didn't consider this.